numpy/core/_ufunc_config.py

```diff
@@ -207,13 +219,17 @@ def setbufsize(size):
 @set_module('numpy')
 def getbufsize():
     """
-    Return the size of the buffer used in ufuncs.
+    Return the size of the buffer used in :ref:`ufuncs`.
 
     Returns
     -------
     getbufsize : int
         Size of ufunc buffer in bytes.
-
+    
+    Examples
+    --------
+    >>> np.getbufsize()  # doctest: +SKIP
+    8192  # may vary
```

Comment:
```suggestion
    8192  # varies by OS
```

Reply:
It doesn't vary by OS, the variations seem due to the test run itself, I think.  The typing tests are slow and don't run everwywhere, and set the bufsize to 4096 without setting it back.  The 20000 is/was maybe also the doc tests running twice or so?

I am not sure when to modify buffer sizes best.  There is an older issue I saw recently, that suggested we should just make them generally smaller, also to better use CPU caches (I guess times change)...

Reply:
I realize I don't know enough about the use cases of these functions to meaningfully improve the docs atm. If I manage to find an example that demonstrates the impact on performance, I'll open another PR. Feel free to close this one, unless you want to discuss this further.